scripts/mkcloud

```diff
@@ -1291,6 +1305,12 @@ function sanity_checks
         fi
     fi
 
+    if [[ $wantedcmds =~ "setupironicnodes" ]]; then
```

Comment:
This check is too late.
Please remember that mkcloud is made to run each step individually, one after the other. We mostly use it with a long list of steps, but manually stepping is supported and used as well.
So this sanity check runs too late in the process.

Rather use the generic proposal enable/disable mechanism: `if [[ $want_ironic_proposal = 1 ]] ; then`